docs/docs/adrs/adr-015-partial-set-security.md

```diff
@@ -42,25 +51,28 @@ For example, `53` corresponds to a Top 53% chain, meaning that the top `53%` pro
 `top_N`  can be `0` or include any value in `[50, 100]`. A chain can join with `top_N == 0` as an Opt In, or with `top_N ∈ [50, 100]` as a Top N chain.
 
 In case of a Top N chain, we restrict the possible values of `top_N` from `(0, 100]` to `[50, 100]`.
-By having `top_N >= 50` we can guarantee that we cannot have a successful invalid-execution attack, assuming that at most `1/3` of provider validators can be malicious.
-This is because, a Top N chain with `N >= 50%` would have at least `1/3` honest validators, which is sufficient to stop invalid-execution attacks.
+By having `top_N >= 50` we can guarantee that we cannot have a successful attack, assuming that at most `1/3` of provider validators can be malicious.
```

Comment:
I think the `>= 50` is rather specific to invalid-execution attacks. For instance, we do not ask for `>= 50` to avoid equivocation attacks. So removing "invalid-execution" might be too general.

Reply:
I agree. Obviously it's true that we care about other attacks, too, but the exact choice of N>=50 rather than N>=66 or whatever is because of what Karolos argued here afaict

Reply:
Maybe a way to resolve this is to leave the original, but then go on to say:
"Choosing N>=50 also prevents other attacks, like chaindeath/liveness attacks." to make clear we also consider other attacks